test/infrastructure/kind/mapper.go

```diff
@@ -79,6 +79,62 @@ type Mapping struct {
 var preBuiltMappings = []Mapping{
 
 	// TODO: Add pre-built images for newer Kind versions on top
+	// Pre-built images for Kind v1.22.
```

Comment:
@fabriziopandini I did comment to remove them because they did contain the same kubernetes versions as for kind v0.22 and would never get used because of the kind v0.22 images would match first.

Reply:
I prefer to keep the bump mechanical (we should not check sha manually, just add them in order)

Reply:
Talked to fabrizio: let's re-add the images (sorry for aksing you to remove it earlier) :-)

Reply:
np!  just added them back in when you have a chance to review